### ISSUE TITLE
Invalid URL TypeError

### ISSUE THREAD

**Issue by rasos** [citizenos/ep_image_upload#33]
Since a plugin update today to ep_image_upload@1.0.32 and ep_image_upload@1.0.33 we get on a EP 1.8.7 at each upload: 

```
[2021-02-14 23:19:48.971] [ERROR] console - TypeError [ERR_INVALID_URL]: Invalid URL: testing1/ba1e83de-1b80-427a-9ce8-2c6780076.jpeg
    at new NodeError (node:internal/errors:329:5)
    at onParseError (node:internal/url:282:9)
    at new URL (node:internal/url:358:5)
    at new URL (node:internal/url:355:22)
    at Busboy.<anonymous> (/opt/etherpad-lite/node_modules/ep_image_upload/index.js:132:24)
    at Busboy.emit (node:events:378:20)
    at Busboy.EventEmitter.emit (node:domain:470:12)
    at Busboy.emit (/opt/etherpad-lite/node_modules/busboy/lib/main.js:38:33)
    at PartStream.<anonymous> (/opt/etherpad-lite/node_modules/busboy/lib/types/multipart.js:213:13)
    at PartStream.emit (node:events:378:20)
    at PartStream.EventEmitter.emit (node:domain:470:12)
    at HeaderParser.<anonymous> (/opt/etherpad-lite/node_modules/dicer/lib/Dicer.js:51:16)
    at HeaderParser.emit (node:events:378:20)
    at HeaderParser.EventEmitter.emit (node:domain:470:12)
    at HeaderParser._finish (/opt/etherpad-lite/node_modules/dicer/lib/HeaderParser.js:68:8)
    at SBMH.<anonymous> (/opt/etherpad-lite/node_modules/dicer/lib/HeaderParser.js:40:12)
[2021-02-14 23:19:48.971] [INFO] console - Stopping Etherpad...
```
`settings.json` includes:

```
"ep_image_upload": {
    "storage": {
      "type": "local",
      "baseFolder": "/mnt/data/etherpad-lite/upload",
      "baseURL": "https://ourpad.net/up/"
    },
    "fileTypes": ["jpeg","jpg","bmp","gif","png"],
    "maxFileSize": 5000000
},
```

This has worked fine before. Uninstalling the plugin for now, as it causes exceptions.

**Comment by JohnMcLear** [comment]
Today I only introduced logic for increasing socketio max message size through plugin afaik.

Do you get this issue on develop?  Either way I will fix but it's weird you get this issue since it's not related to socketio.

**Comment by JohnMcLear** [comment]
Oh I see this issue.  It's not related to today's work.  The issue is 5 or so days old.   I will fix ASAP.

**Comment by JohnMcLear** [comment]
Oh, I just spotted that you serve files through nginx/apache or something and not from Etherpad.  Which is fine, it means I need to mess w/ my config a bit more but no big deal :)

**Comment by JohnMcLear** [comment]
I pushed a blind fix, feel free to test.

**Comment by JohnMcLear** [comment]
Ugh, I'm not entirely sure this will work on 1.8.8...  I just tested and I don't think the socketio msg size is being respected.....

**Comment by JohnMcLear** [comment]
I think having test coverage for actually uploaded files would be handy, but it's beyond the scope of my time atm

**Comment by JohnMcLear** [comment]
If this is still an issue in latest release please comment

**Comment by rasos** [comment]
Fix tested on EP 1.8.7. Works here, thanks for the quick response.

**Comment by JohnMcLear** [comment]
Yw plz update to 1.8.8 :) tbx
